### DISCUSSION TITLE
Any plans to support description and external docs for the tag decorator?

### DISCUSSION THREAD

**Discussion by victorseara** [microsoft/typespec#4802] [Q&A] [answered]
I've been using Redoc to render an OpenApi spec, and it would be nice if typespec could output the additional tag properties to the generated spec. 

Currently, the tag decorator accepts only a `string`, but OAS 3 supports `description` and `externalDocs` in the root tag array (both are optional).

Are there any plans to support those properties? I noticed the decorator is exported from the standard library, so it may have other use cases I'm unaware of, but it would be a nice addition for this specific use case.

**Comment by timotheeguerin** [comment, marked as answer]
We have an issue https://github.com/microsoft/typespec/issues/2220. I believe someone was going to work on it soon.
